src/components/FavorizeButton/FavorizeButton.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+FavorizeButton.propTypes = {
+  isFavorite: PropTypes.bool.isRequired,
+  onToggleFavorite: PropTypes.func.isRequired,
+  id: PropTypes.number.isRequired,
+}
+
+export default function FavorizeButton({ isFavorite, onToggleFavorite, id }) {
+  return (
+    <Button data-testid="favorize-button" onClick={() => onToggleFavorite(id)}>
+      <SVG
+        data-testid="svg"
```

Comment:
Have a look at the possible queries: https://testing-library.com/docs/dom-testing-library/cheatsheet/#queries

You might want to consider giving your [svg a title](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) and then try to query by that. Would also enhance accessibility 🙂